common/repo/GitRepo.java

```diff
@@ -118,18 +114,29 @@ private CommandResult runCommand(String... command) {
     if (!result.stderr.isEmpty()) {
       throw new RuntimeException(formatError(result));
     }
+    commandLog.add(result);
     return result;
   }
 
   private String formatError(CommandResult commandResult) {
-    return String.format(
+    StringBuilder result = new StringBuilder();
+    result.append(String.format(
         "\n%s\n%s",
         commandResult.command,
-        commandResult.stderr);
+        commandResult.stderr));
+    if (commandLog.size() > 0) {
+      result.append("Previous git commands (most recent is on top):\n");
+      for (CommandResult previousCommand : Lists.reverse(commandLog)) {
```

Comment:
`previousCommand` variable is never used.

Reply:
Thanks!